### PR TITLE
refactor: remove duplicate header data from notification register body

### PR DESCRIPTION
**Pull request by rosvik**
closes https://github.com/AtB-AS/kundevendt/issues/15875

### CODE REVIEW

File: src/notifications/api/index.ts [hunk 1]
```diff
@@ -19,14 +19,18 @@ export const registerForPushNotifications = ({
...
-        platform: Platform.OS,
-        platform_version: String(Platform.Version),
         language,
         token,
       },
-      {authWithIdToken: true},
+      {
+        authWithIdToken: true,
+        headers: {
+          'atb-app-identifier': getBundleId(),
```
[the remark this answers is not in the document]
Comment (KristianSelnas) [reply src/notifications/api/index.ts]: Yes, they are. It was solved here https://github.com/AtB-AS/mittatb-app/pull/4019

**Review by KristianSelnas — COMMENTED** [review]
I don't think this is necessary.

**Review by KristianSelnas — APPROVED** [review]
You should rename the PR in order for the commit message to reflect what's actually inluded.

**Review by gorandalum — APPROVED** [review]

File: src/notifications/api/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (tormoseng) [reply src/notifications/api/index.ts]: Nice! Then look away from my comment :-)

**Pull request merged**